onnx/tests/onnx/mod.rs

```diff
@@ -121,6 +119,8 @@ pub fn run_one<P: AsRef<path::Path>>(root: P, test: &str, optim: bool) {
                 );
             }
             for (ix, (a, b)) in computed.iter().zip(expected.iter()).enumerate() {
+//                println!("computed: {:?}", computed[ix].dump(true));
```

Comment:
Temporary comments?

Reply:
Well this is test code, they come in handy when something does not work. I know it looks a bit dirty, but I haven't find a much better way.